plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -850,4 +850,491 @@ AND rnd_uuid IN (UUID '1fc74d96-0216-449b-a145-455578a9eaa5', UUID '3ee49ede-002
 
         assertUpdate("DROP TABLE faker.default.all_types_in");
     }
+
+    @Test
+    void testSelectRangeProperties()
+    {
+        @Language("SQL")
+        String tableQuery;
+        @Language("SQL")
+        String testQuery;
+
+        // inclusive ranges that produce only 2 values
+        // high boundary float value obtained using `Math.nextUp((float) 0.0)`
+        tableQuery =
+                """
+                CREATE TABLE faker.default.all_types_range_prop (
```

Comment:
can you change code structure so column definition and expected value are side by side (a list of records?).
And generate SQL from that. 
With this shape it is hard to match what is going on.
Also it may make sense to generate separate table with single column for each column definition you want to test.

Reply:
Done! But as a separate commit.